src/riscv/lib/src/machine_state/page_cache/interpreted.rs

```diff
@@ -29,6 +29,8 @@ pub struct InterpretedCompiler;
 #[derive(derive_more::Debug)]
 pub struct Interpreted<MC, M> {
     instruction: Instruction,
+    #[cfg(test)]
+    call_count: std::cell::RefCell<usize>,
```

Comment:
A `Cell` should be sufficient.

```suggestion
    call_count: std::cell::Cell<usize>,
```